transfer-data/flows.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Flows'
+description: 'This page explains how to set up a flow to transfer and transform data from an Axiom dataset to a destination.'
+---
+
+import Prerequisites from "/snippets/minimal-prerequisites.mdx"
+
+<Note>
+Flow is currently in preview. To try it out, [sign up for a free preview](https://forms.axiom.co/flow).
+</Note>
+
+A flow is a way to transfer and transform data from an Axiom dataset to a destination that you choose. This page explains how to set up a flow.
+
+<Prerequisites />
+{/* list separator */}
+- Set up a destination. For more information, see [Destinations](/transfer-data/destinations).
+
+## Set up a flow
+
+To set up a flow:
+1. Click the **Flows** tab. Axiom displays the list of flows you have created.
+2. In the top right, click **New flow**.
+3. In the **Source** section, specify the source dataset and the transformation in an APL query. For example, the following APL query selects events from the `cloudflare-logpush` dataset and transforms the data by deleting and adding a field.
+
+    ```kusto
+    // Transfer high-value app events to a costly SIEM
+    // while retaining the complete record of all events in Axiom.
```

Comment:
```suggestion
```

I would remove this tbh. I don’t think it makes so much sense in this context. Better described in the numbered bullet above if wanting to include imo.